models/oscr-extract.js

```diff
@@ -0,0 +1,72 @@
+
+
+// Using th schema from the CSV download available at http://www.oscr.org.uk/charities/search-scottish-charity-register/charity-register-download
+var schemas = { "v0.1": {} };
+
+schemas["v0.1"].oscr_register = {
+  // registered number of a charity
+  "Charity Number": String,
```

Comment:
Yep, agreed on spaces in names. The reason I kept them was because they match with the CSV headers so you can just import each CSV row object - but it shouldn't be too difficult to change.